p2pv2/stream_pool.go

```diff
@@ -230,7 +227,9 @@ func (sp *StreamPool) sendMessageWithResponse(ctx context.Context, msg *p2pPb.Xu
 	}
 	res, err := str.SendMessageWithResponse(ctx, msg)
 	if err != nil {
-		sp.DelStream(str)
+		if common.NormalizedKVError(err) == common.ErrP2PError {
```

Comment:
NormalizedKVError is not a good name for ErrP2PError.
ErrP2PError and the Normalized function should be in p2pv2 package?